pkg/controller/volume/persistentvolume/testing/testing.go

```diff
@@ -113,12 +111,6 @@ func (r *VolumeReactor) React(action core.Action) (handled bool, ret runtime.Obj
 			return true, nil, fmt.Errorf("Cannot create volume %s: volume already exists", volume.Name)
 		}
 
-		// mimic apiserver defaulting
-		if volume.Spec.VolumeMode == nil && utilfeature.DefaultFeatureGate.Enabled(features.BlockVolume) {
```

Comment:
This is removed because defaulting is now on by default?

Reply:
Hmm, it is defaulted by API server, but this is unit test... I guess all VolumeMode users correctly interpret `nil` as `Filesystem`.

I restored this chunk to really mimic API server defaulting and removed feature gate check.